foundations/azure/referenceImplementations/core/managementGroupTemplates/policyDefinitions/Compliant-MySQLPolicySetDefinition.json

```diff
@@ -17,47 +17,538 @@
+                            "metadata": {
+                                "version": "1.0.0",
+                                "category": "MySQL"
+                            },
+                            "parameters": {
+                                "effect": {
+                                    "type": "string",
+                                    "defaultValue": "Deny",
+                                    "allowedValues": [
+                                        "AuditIfNotExists",
+                                        "Disabled",
+                                        "Deny"
+                                    ],
+                                    "metadata": {
+                                        "displayName": "Effect",
+                                        "description": "Enable or disable the execution of the policy"
+                                    }
+                                }
+                            },
+                            "policyRule": {
+                                "if": {
+                                    "field": "type",
+                                    "equals": "Microsoft.DBforPostgreSQL/servers"
+                                },
+                                "then": {
+                                    "effect": "[parameters('effect')]",
+                                    "details": {
+                                        "type": "Microsoft.DBforPostgreSQL/servers/administrators"
+                                    }
+                                }
+                            }
                         }
+                    },
+                    "name": "Deny-MySql-Without-Aad-Admin"
+                },
+                {
+                    "properties": {
+                        "properties": {
+                            "displayName": "Private endpoint should be enabled for MySQL servers",
+                            "policyType": "BuiltIn",
+                            "mode": "Indexed",
+                            "description": "Private endpoint connections enforce secure communication by enabling private connectivity to Azure Database for MySQL. Configure a private endpoint connection to enable access to traffic coming only from known networks and prevent access from all other IP addresses, including within Azure.",
+                            "metadata": {
+                                "version": "1.0.3",
+                                "category": "MySQL"
+                            },
+                            "parameters": {
+                                "effect": {
+                                    "type": "string",
+                                    "defaultValue": "AuditIfNotExists",
+                                    "allowedValues": [
+                                        "AuditIfNotExists",
+                                        "Disabled",
```

Comment:
In general: If existing policyEffect is auditIfNotExist (resourceType existence and config), there's no direct translation to Deny (properties within the existing resourceType). This would also be challenging to assign at scale due to the privateEndpointId required.